transport/tchannel/header.go

```diff
@@ -98,15 +98,18 @@ func writeRequestHeaders(
 	format tchannel.Format,
 	appHeaders map[string]string,
 	getWriter func() (tchannel.ArgWriter, error),
+	writeExactHeaderCase bool,
 ) error {
+	if writeExactHeaderCase {
+		return writeHeaders(format, appHeaders, getWriter)
```

Comment:
why do we make a copy of headers below, but not in the `writeExactHeaderCase`?

Reply:
I believe the sole purpose of this 106-110 block is to canonicalize the headers. The `Items` will return the underlying map that is not subject to modification, which is not exactly an Copy.

Reply:
I don't understand -- the headers are already canonicalized right? Or this is to canonicalize the Jaeger headers?

Reply:
it is to canonicalize the Jaeger headers